components/__snapshots__/Layout.test.tsx.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
All the `__snapshots__` folders must be ignored. Could you please add this in the .gitignore?